comms/dht/migrations/2020-05-14-103121_add_body_hash/down.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE stored_messages
+    DROP COLUMN body_hash;
```

Comment:
Need to add add the counterpart to `CREATE UNIQUE INDEX` (i.e. DROP INDEX) above ALTER TABLE, otherwise I believe it'll error(not sure) on the down migration

Reply:
Change applied.